Stripe/StripeiOSTests/STPPaymentMethodSunbitParamsTests.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  STPPaymentMethodSunbitParamsTests.swift
+//  StripeiOSTests
+//
+//  Created by Eric Geniesse on 6/27/24.
+//
+
+import Foundation
+import StripeCoreTestUtils
+
+@testable@_spi(STP) import Stripe
+@testable@_spi(STP) import StripeCore
+@testable@_spi(STP) import StripePayments
+@testable@_spi(STP) import StripePaymentSheet
+@testable@_spi(STP) import StripePaymentsUI
+
+class STPPaymentMethodSunbitParamsTests: XCTestCase {
+
+    func testCreateSunbitPaymentMethod() throws {
+        let sunbitParams = STPPaymentMethodSunbitParams()
+
+        let params = STPPaymentMethodParams(
+            sunbit: sunbitParams,
+            billingDetails: nil,
+            metadata: nil
+        )
+
+        let exp = expectation(description: "Payment Method Sunbit create")
+
+        let client = STPAPIClient(publishableKey: STPTestingDefaultPublishableKey)
+        client.createPaymentMethod(with: params) {
+            (paymentMethod: STPPaymentMethod?, error: Error?) in
+            exp.fulfill()
+
+            XCTAssertNil(error)
+            XCTAssertNotNil(paymentMethod, "Payment method should be populated")
+            XCTAssertEqual(paymentMethod?.type, .sunbit, "Incorrect PaymentMethod type")
+            XCTAssertNotNil(paymentMethod?.sunbit, "The `sunbit` property must be populated")
```

Comment:
This looks entirely redundant with the test in STPPaymentMethodFunctionalTest except for this line. Why did we start doing this..

Let's delete this file and add this line to `STPPaymentMethodFunctionalTest`.

Not a huge win but we should probably do the same for all the other "Params" tests that actually just create PMs :/ @davidme-stripe

Reply:
Overly pedantic unit tests, probably. They date to the beginning of the SDK

Reply:
Done!